src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -554,20 +554,24 @@ def _resolve_python_flow_additional_includes(source) -> Path:
 
 
 def serve_flow_python(args, source):
+    from promptflow._sdk._configuration import Configuration
     from promptflow.core._serving.app import create_app
 
     static_folder = args.static_folder
     if static_folder:
         static_folder = Path(static_folder).absolute().as_posix()
     config = list_of_dict_to_dict(args.config)
+    pf_config = Configuration(overrides=config)
```

Comment:
Configuration is still in SDK?

Reply:
Yes, this is CLI file, it's ok to import SDK